<launch>
<arg name="verbose" default="false"/>
<arg name="generate_points" default="true"/>
<!--arg name="cameras" default="[0,]" /-->
<arg name="sensor_model" default="helios"/>
<arg name="voxel_leaf" value="0.01" />
<arg name="rviz_name" default="unet.rviz"/>

<node name="evalcloud"  pkg="ros_unet" type="ros_evalcloud.py" output="screen" required="true">

</node>


</launch>

